core/src/main/java/io/grpc/NameResolver.java

```diff
@@ -350,4 +408,143 @@ public String toString() {
       }
     }
   }
+
+  /**
+   * Represents the results from a Name Resolver.
+   *
+   * @since 1.20.0
+   */
+  @ExperimentalApi("FIXME")
```

Comment:
You can use `NameResolver`'s tracking issue: https://github.com/grpc/grpc-java/issues/1770

Reply:
done.